dataRetriever/dataPool/sovereign/outGoingBridgeOperations.go

```diff
@@ -0,0 +1,30 @@
+package sovereign
+
+import (
+	"context"
+
+	"github.com/multiversx/mx-chain-core-go/data/sovereign"
+)
+
+type outGoingBridgeOperations struct {
+}
+
+// NewOutGoingBridgeOperationsHandler creates a new sovereign outgoing bridge operations handler
+func NewOutGoingBridgeOperationsHandler() (*outGoingBridgeOperations, error) {
+	return &outGoingBridgeOperations{}, nil
+}
+
+// Send should be able to send bridge data operations from sovereign
+func (op *outGoingBridgeOperations) Send(ctx context.Context, data *sovereign.BridgeOperations) error {
```

Comment:
func (op `*outGoingBridgeOperations)` Send(_ context.Context, _ *sovereign.BridgeOperations) error {

and this part can be removed 
```
_ = ctx
_ = data
```